codemodder/codemods/api/__init__.py

```diff
@@ -44,7 +46,7 @@ def __init_subclass__(cls):
         # But it is necessary to get around the fact that these fields are
         # checked by __init_subclass__ of the other parents of SemgrepCodemod
         # first.
-        if not cls.__name__ == "SemgrepCodemod":
+        if cls.__name__ not in ("BaseCodemod", "SemgrepCodemod"):
```

Comment:
This works for now but I do view this whole thing as a workaround for the time being.